### PR TITLE
Fix Challenge Numbers

### PR DESCRIPTION
**Pull request by Jayashsatolia403**
## Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/0chain/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

## Associated PRs (Link as appropriate):
- blobber:
- gosdk:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

File: code/go/0chain.net/smartcontract/storagesc/challenge_eventdb.go [hunk 1]
```diff
@@ -57,73 +58,63 @@ func challengeTableToStorageChallengeInfo(ch *event.Challenge, edb *event.EventD
...
 		AllocationID:   sc.AllocationID,
 		BlobberID:      sc.BlobberID,
-		Responded:      sc.Responded,
 		RoundResponded: balances.GetBlock().Round,
 		Passed:         passed,
 	}
+	if passed {
+		clg.Responded = int64(1) // Passed challenge
+	} else {
+		clg.Responded = int64(2) // Failed challenge
```
Comment (dabasov) [inline code/go/0chain.net/smartcontract/storagesc/challenge_eventdb.go:92]: why 2?

File: code/go/0chain.net/smartcontract/storagesc/challenge_eventdb.go [hunk 2]
```diff
@@ -57,73 +58,63 @@ func challengeTableToStorageChallengeInfo(ch *event.Challenge, edb *event.EventD
...
+		OpenChallenges:      uint64(blobberStats.OpenChallenges),
 	}
 
-	if passed {
-		a.SuccessfulChallenges = 1
-		b.ChallengesPassed = 1
-	} else {
-		a.FailedChallenges = 1
-	}
+	logging.Logger.Info("jayash A ", zap.Any(
```
Comment (dabasov) [inline code/go/0chain.net/smartcontract/storagesc/challenge_eventdb.go]: please remove logging

**Review by dabasov — CHANGES_REQUESTED** [review]

File: code/go/0chain.net/smartcontract/storagesc/challenge_eventdb.go (hunk 1 above)
Comment (Jayashsatolia403) [reply code/go/0chain.net/smartcontract/storagesc/challenge_eventdb.go:92]: For expired it is 2, for open it is 0 and for passed it is 1

**Review by dabasov — APPROVED** [review]
lgtm

**Pull request merged**